edge-functions/crypto/package.json

```
@@ -4,21 +4,21 @@
   "repository": "https://github.com/vercel/examples.git",
   "license": "MIT",
   "scripts": {
-    "dev": "next dev",
     "build": "next build",
-    "start": "next start",
-    "lint": "next lint"
+    "dev": "next dev",
+    "lint": "next lint",
+    "start": "next start"
   },
   "dependencies": {
-    "next": "^12.1.0",
+    "next": "12.1.7-canary.45",
```

Let's use `canary` here, as those versions change quite fast and we want every example to always be on the latest canary if there are new improvements on the way.

```suggestion
    "next": "canary",
```

---

It doesn't work with canary. It points to another version. Has to be a fixed version for now.